src/integrationTest/java/uk/gov/hmcts/darts/event/testutils/IntegrationBase.java

```diff
@@ -2,7 +2,9 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
 
+@AutoConfigureWireMock(port = 8070)
```

Comment:
Great, I was going to suggest this to reduce the boilerplate needed in each of our int tests.